src/AppBundle/Entity/Post.php

```diff
@@ -95,7 +95,7 @@ class Post
      *      mappedBy="post",
      *      orphanRemoval=true
      * )
-     * @ORM\OrderBy({"publishedAt" = "DESC"})
+     * @ORM\OrderBy({"publishedAt": "DESC"})
```

Comment:
Good spot! I think `:` is more native since it looks like a JSON object here, so I like it `:` more. But if both works, I'd rather send a PR to Doctrine docs to use `:` instead of `=` and see what they'll say :)